### PR TITLE
Weekly forecast, autocomplete search of locations, theme switcher, redux

### PR DESCRIPTION
**Pull request by makeychik**

### CODE REVIEW

File: src/components/Autocomplete/Autocomplete.jsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React, { useEffect } from 'react';
+import AutocompleteOption from '../AutocompleteOption/AutocompleteOption';
+import classes from './Autocomplete.module.scss';
+
+const Autocomplete = ({ display, options, onOptionSelect }) => {
+  // useEffect(() => {}, [display, options]);
```
Comment (JuliaShavyrova) [inline src/components/Autocomplete/Autocomplete.jsx]: please remove comments

Comment (makeychik) [reply src/components/Autocomplete/Autocomplete.jsx]: done

Comment (JuliaShavyrova) [reply src/components/Autocomplete/Autocomplete.jsx]: please remove commented code everywhere you have it (if it won't be used anymore)

File: src/components/DailyForecastItem/DailyForecastItem.test.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+import DailyForecastItem from './DailyForecastItem';
+
+const forecastItem = {
+  time: '2021-12-20T13:00+04:00',
+  symbol: 'd400',
+  temperature: -0.78
+};
+
+describe('DailyForecastItem component', () => {
+  it('component rendered successfully and match the snapshot', () => {
```
Comment (JuliaShavyrova) [inline src/components/DailyForecastItem/DailyForecastItem.test.js:12]: would be great to have more test cases, not only test by snapshot

**Review by JuliaShavyrova — APPROVED** [review]

**Pull request merged**